wgengine/magicsock/magicsock.go

```diff
@@ -89,6 +90,8 @@ type Conn struct {
 	// ============================================================
 	mu sync.Mutex // guards all following fields
 
+	canCreateEPUnlocked syncs.AtomicBool // whether mu is held; for unfortunate CreateEndpoint use
```

Comment:
Can you make this a block comment rather than inline, and expand on what's unfortunate? The current comment scares me without informing me.

Reply:
Without having seen how this is used yet, also seems like it wants to be above mu in the code, because mu "guards all following fields", suggesting you can't touch this AtomicBool at all without holding mu, which seems nonsensical.

Reply:
Updated comment.